server/middleware/forceHTTPS.js

```diff
@@ -0,0 +1,12 @@
+function forceHTTPS(req, res, next) {
+
+    if (process.env.FORCE_HTTPS === 'true' && !req.secure) {
```

Comment:
food for thought: I often like to keep a "constants" file handy that loads all the environment variables into a single entrypoint so they are easy for newcomers of the code to discover. Consider making a `constants.js` file to extract `process.env` and importing into here